src/components/molecules/storage/FileListItem.tsx

```diff
@@ -60,6 +61,13 @@ const FileListItem = ({ file, arweave, arweaveWallet }: FileListItemProps) => {
           onClick={() => addToArweave(sCid, file.name)}
           size="sm"
         ></IconButton>}
+        {ipfsNode && <IconButton
+            variantColor="teal"
+            icon="check"
```

Comment:
a "check" icon imo doesn't communicate the action good enough. I'd vote for `attachment` here to keep things simple

Reply:
changed